scripts/ci/unit_test.sh

```diff
@@ -16,29 +16,31 @@ report_coverage_prow() {
     BUILD_NUMBER="" PULL_REQUEST_NUMBER=${PULL_NUMBER} goveralls \
            -coverprofile=$COVER_PROFILE \
            -service=prow \
-           -repotoken $COVERALLS_TOKEN
+           -repotoken "$COVERALLS_TOKEN"
 }
 
 report_coverage_travis() {
     goveralls -coverprofile=$COVER_PROFILE \
            -service=travis-ci \
-           -repotoken $COVERALLS_TOKEN
+           -repotoken "$COVERALLS_TOKEN"
 }
 
 echo Running tests:
-go test -covermode=count -coverpkg=./pkg/... -coverprofile=$COVER_PROFILE.tmp ./pkg/...
-cat $COVER_PROFILE.tmp | grep -v "zz_generated" > $COVER_PROFILE
+# tests with negated `unittests` build tag will not be ran
+go test -tags=unittests -covermode=count -coverprofile=$COVER_PROFILE.tmp ./apis/... ./controllers/... ./pkg/...
```

Comment:
Introduced the `unittests` build tag. Added a negated `unittests` tag to the files which shouldn't be run along with unit tests (2 integration envtest suites)